### ISSUE TITLE
Message Board

### ISSUE THREAD

**Issue by eadward** [amresearchlab/polyhymnia-listenerprofile#1]
Hi! Any news on the project?

**Comment by Lewis-lv0** [comment]
Yeah, definitely. I am now preparing a demo for my major updates :-)
I have finished implementing login/register features and merging them with your frontend. I am now working on the Spotify API and the user profile page. Specifically, I use a database to store and retrieve data. So I need to change many csv staffs to an integrated database. In addition, we may add other music streams apart from Spotify. I take it into consideration and reorganised the design of the backend.

**Comment by eadward** [comment]
That's great. Once you have something for me to look at let me know. Then we can arrange to talk.

**Comment by Lewis-lv0** [comment]
Hi Professor! I have completed my first demo. Descrptions and my work progress are shown in [DOCUMENTATION](https://github.com/amresearchlab/polyhymnia-listenerprofile/blob/main/README.md#documentation). More details are in [demo1](https://github.com/amresearchlab/polyhymnia-listenerprofile/tree/main/demos/demo1). For the next step, I will work on the questionnaires.

**Comment by eadward** [comment]
Hi! Sorry but things have been a bit crazy and just this now. Will check ASAP? Were there any changes in the meantime?

**Comment by Lewis-lv0** [comment]
These days I am thinking about a front-end tool to replace Dash. Maybe React+antv [](https://antv.vision/en) will be a good choice. I hope we can have a meeting to talk about the project at your convenience :)

**Comment by eadward** [comment]
Hi, check the app but I had problems linking to Spotify so could test. All seems ok anyway. Looking forward to see the questionnaires working and data being retrieved and saved. If you have time we can meet next week. Tuesday at 11am is OK for you? If not suggest another time. What's you Uni email address?

**Comment by Lewis-lv0** [comment]
Hi Professor! I am not available at that time because I have an on-campus tutorial. A few days ago I talked with Xiaoyuan and we suggest to have a meeting with you together to talk about our projects. He will be available 10 days later because he needs to stay in quarantine after his arrival. We will check our timatable and suggest a time ASAP. These days I am working on the questionnaires and data retrieval. I will also prepare a demo to show you at the meeting. 
My Uni email address is sgylu27@liverpool.ac.uk

**Comment by eadward** [comment]
That's perfect. Hope all is going well with the start of the semester.

**Comment by Lewis-lv0** [comment]
Hi Professor! We have checked our timetable. We are available at Oct. 11th, 12th and 13th after 2 p.m. I hope you can pick a time slot at your convenience and then we can have a meeting :)

**Comment by eadward** [comment]
Oct 13 at 2.30 pm should be ok. will send an outlook invitation to meet in person now. If you prefer Zoom let me know.
